### PR TITLE
Version 3.2.0 Changelog Prep

### PR DESCRIPTION
**Pull request by bwieger-atlassian-com**
Adds Changelog in preparation for release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## What's New in 3.2.0
+
+### Improvements 
+
+- PAT authentication is now supported for JIRA.
```
Comment (sdzh-atlassian) [inline CHANGELOG.md:5]: Might be a silly question - but does it actually work for JIRA cloud? Or is it exclusively a server thing?

Comment (sdzh-atlassian) [reply CHANGELOG.md:5]: Found the answer - it's [only DC and Server](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html). Perhaps we could clarify here?
image

Comment (bwieger-atlassian-com) [reply CHANGELOG.md:5]: I think it should work for the cloud. We can give it a test. 

Cloud supports PAT.

**Review by sdzh-atlassian — APPROVED** [review]
SHIP IIIIT 🚢

**Pull request merged**